tests/test_linalg/test_solvers/cases/states.py

```diff
@@ -1,25 +1,46 @@
 """Probabilistic linear solver state test cases."""
 
 import numpy as np
+from pytest_cases import case
 
 from probnum import linalg, linops, randvars
 from probnum.problems.zoo.linalg import random_linear_system, random_spd_matrix
 
+# Problem
+n = 10
+linsys = random_linear_system(
+    rng=np.random.default_rng(42), matrix=random_spd_matrix, dim=n
+)
 
-def case_linear_solver_state(
+# Prior
+prior = linalg.solvers.beliefs.LinearSystemBelief(
+    A=randvars.Constant(linsys.A),
+    Ainv=None,
+    x=randvars.Normal(
+        mean=np.zeros(linsys.A.shape[1]), cov=linops.Identity(shape=linsys.A.shape)
+    ),
+    b=randvars.Constant(linsys.b),
+)
```

Comment:
Do we want to make fixtures out of these?

Reply:
I'll leave them as cases for now and ina future PR go over the test structure again to see whether it makes sense to have fixtures for those objects that are not compositional (and thus depend on other cases), or whether we just build up everything from cases.